pages/api/utils/policy-utils.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const mapToClientModel = (policyResponse) => {
+  let formattedResponse = policyResponse;
+
+  if (typeof policyResponse === "string") {
```

Comment:
If there are better ways to handle this, I am very open to suggestions. It feels weird to be checking like this, but this was the most obvious solution to me and works for our current use cases.

Reply:
I missed where `mapToClientModel` is called with a string. It looks like it's always called after `response.json`, is there some double-encoded output from Rode?

I can see `mapToApiModel` is called with the request body, which makes sense.

Reply:
Ooo yeah, you're right; this isn't super relevant to `mapToClientModel`, I probably should have left this comment in the other function. I can't think of a reason to keep this check in here so I can remove it.